src/components/LetterSketchExample.js

```diff
@@ -0,0 +1,212 @@
+  constructor() {
+    super();
+    this.state = {
+      points: []
+    };
+
+    // TODO: get letter from props
+    // TODO: create letter target points generation algorithm if the given way
+    // of checking an answer will be fine
+    this.letter = [
+      [79, 415],
+      [78.5, 388],
+      [79.5, 351.5],
+      [81.5, 324],
+      [88, 292],
+      [93, 253],
+      [103, 214.5],
+      [118, 183],
+      [140, 165],
+      [167, 152.5],
+      [201.5, 147.5],
+      [228.5, 152.5],
+      [252, 182.5],
+      [267.5, 220],
+      [280, 255.5],
+      [286.5, 295],
+      [288.5, 335.5],
+      [288, 365.5],
+      [288.5, 403.5],
+      [288.5, 429.5],
+      [110, 300.5],
+      [139.5, 298.5],
+      [168.5, 300.5],
+      [201, 300],
+      [232, 301.5],
+      [264.5, 301]
+    ];
+
+    // settings
+    // NOTE: these are values in pixels, make it depend on the phone resolution?
+    this.strokeWidth = 30;
+    this.minThreshold = 20;
+    this.maxThreshold = 20;
+    this.passThreshold = 0.8;
+  }
+
+  componentWillMount() {
+    this._panResponder = PanResponder.create({
+      onStartShouldSetPanResponder: () => true,
+      onPanResponderGrant: (evt, gestureState) => {
+        const points = this.state.points;
+        points.push([]);
+        points[points.length - 1].push([gestureState.x0, gestureState.y0]);
+        this.setState({ points });
+      },
+      onPanResponderMove: (evt, gestureState) => {
+        const points = this.state.points;
+        points[points.length - 1].push([
+          gestureState.moveX,
+          gestureState.moveY
+        ]);
+        this.setState({ points });
+        // NOTE: it would be better to directly change the component's path
+        // value, instead of calling setState every time,
+        // but there's some problem with this in the project (haven't figured it out yet)
+        // (it can be either the library itself or something to do with expo,
+        // but the same example works with the default react-native project):
+        // this.path - ref of the component
+        // this.path.setNativeProps({ points: this.generatePoints(points) })
+      },
+      onPanResponderRelease: (evt, gestureState) => {}
+    });
+  }
+
+  // NOTE: it's better to append new value to given path during movement
+  // and keep paths to prevent recalculation every time
+  generateTextPoints = path => {
+    const textPoints = [];
+    for (let i = 0; i < path.length; i++) {
+      const point = path[i];
+      textPoints.push(point.join(','));
+    }
+    return textPoints.join(' ');
+  };
+
+  // @checkAnswer
+  // letter - target points which form letter
+  // points - user points [[[x, y], [x, y]]], contains paths, each path contains points
+  // minThreshold - min distance between target point and closest user point
+  //    to consider target point as "passed"
+  // maxThreshold - max distance between any closest user point and target point
```

Comment:
Nice, I like that. Probably will have to review the code to understand it completely, but I like that approach!